### PR TITLE
Update roles-and-teams.md

### PR DESCRIPTION
**Pull request by pixelwake**
Final step in #124.

Updated to reflect the new priority roles.

Closing PR #167 as content changes are now captured here.

@fureigh ready for review.

### CODE REVIEW

File: pages/roles-and-teams.md [hunk 1]
```diff
@@ -2,60 +2,21 @@
...
-### Comms Team
-
-**Director of Outreach**
-
-The Director of Outreach drives a comprehensive internal and external communications strategy for 18F. This includes editorial and content on our website, social media, events/conferences, messaging, and general community evangelism.
+### Communications Team
+[**Director of Outreach**](https://pages.18f.gov/joining-18f/roles-and-teams/positions/director-of-outreach/)
 
 ### Consulting Team
+**Design and Product Strategist** *(Role profile coming soon)*
```
Comment (fureigh) [inline pages/roles-and-teams.md:11]: "Role profile" is jargony and inconsistent with the way that the Director of Outreach position is listed. Also, this approach is overriding the site's means of dynamically displaying open positions. Merging for now and will make a follow-up ticket.

File: pages/roles-and-teams.md [hunk 2]
```diff
@@ -2,60 +2,21 @@
...
-**DevOps & Site Reliability Engineer**
-
-DevOps engineers work tirelessly to arm developers with the best tools and ensure system uptime and performance, helping to bridge gaps with legacy development or operations teams.
-
-**Product Lead**
-
-18F product leads are responsible for managing the development, delivery, and continuous improvement of products while establishing and managing our relationship with our partners (clients).
-
-**Software Developer (Front-End, Back-End, Full-Stack)**
+Site Reliability Engineers ensure our platforms and infrastructure systems &mdash; and the applications on top of them &mdash;  are reliable and easy to deploy and maintain. You’ll be involved in designing and implementing systems in a human-centered way, communicating with users, and finding solutions to help people work more smoothly. Along with providing the platform, you seek opportunities and get creative in  architecting solutions.
```
Comment (fureigh) [inline pages/roles-and-teams.md]: @rooneywp The singular vs. plural suggests that we're hiring for _one_ design and product strategist but _multiple_ SREs. Is that accurate?

**Comment by fureigh** [comment]
@rooneywp As written, this PR changes the open roles to be the following and only the following:
- Director of Outreach
- Design and Product Strategist
- Site Reliability Engineer

Is that accurate? If so, once the singular vs. plural question is resolved, this is good to go.

**Comment by pixelwake** [comment]
@fureigh Additional roles were added to the critical needs list and were added to this PR. They now include
- Director of Outreach
- Design and Product Strategist
- Front End Designer
- Product Lead
- Software Developer
- Site Reliability Engineer
- Visual Designer

**Comment by fureigh** [comment]
@rooneywp Looks good to me. Merging!

**Pull request merged**